File: alicloud-visualbasic/MyStack.vb
Imports Pulumi
Imports Pulumi.AliCloud.Oss

Class MyStack
    Inherits Stack

    Public Sub New()
        ' Create an AliCloud resource (OSS Bucket)
        Dim bucket = New Bucket("my-bucket")

        ' Export the name of the bucket
        Me.BucketName = bucket.Id
    End Sub

    <Output>
    Public Property BucketName As Output(Of String)
End Class

File: alicloud-csharp/MyStack.cs
using Pulumi;
using Pulumi.AliCloud.Oss;

class MyStack : Stack
{
    public MyStack()
    {
        // Create an AliCloud resource (OSS Bucket)
        var bucket = new Bucket("my-bucket");

        // Export the name of the bucket
        this.BucketName = bucket.Id;
    }

    [Output]
    public Output<string> BucketName { get; set; }
}
